src/pages/index.js

```diff
@@ -198,112 +183,35 @@ export default () => {
               <Card className="m-4 bg-white">
                 <div className="sm:mx-auto px-auto flex justify-center content-around flex-wrap">
                   {/* L */}
-                  <a
-                    href="https://cloud.google.com/"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    <img
-                      src={GoogleCloud}
-                      style={{ height: "55px" }}
-                      className="px-5 mt-5 partner"
-                      alt="Google Cloud"
-                    />
-                  </a>
-                  <a
-                    href="https://www.brooksource.com/"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    <img
-                      src={Brooksource}
-                      style={{ height: "100px" }}
-                      className="px-5 mt-15 partner"
-                      alt="Brooksource"
-                    />
-                  </a>
-                  <a
-                    href="https://www.northropgrumman.com/careers/"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    <img
-                      src={Northrop}
-                      style={{ height: "100px" }}
-                      className="px-5 mt-0 partner"
-                      alt="Northrop Grumman"
-                    />
-                  </a>
-                  </div>
-                  <div className="sm:mx-auto px-auto flex justify-center content-around flex-wrap mt-5">
-                    <a
-                      href="https://www.centauricorp.com/your-career/"
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      <img
-                        src={Centauri}
-                        style={{ height: "95px" }}
-                        className="px-10 mt-0 partner"
-                        alt="Centauri"
-                      />
-                    </a>
+                  {sponsors_L.map((spl) => (
+                    <SponsorCard key={spl.link} className="m-4 text-secondary bg-white">
+                      <a href={spl.link} target={spl.target} rel={spl.rel}>
+                      <img src={spl.src} style={{ height: spl.imageHeight }} className={spl.className} alt={spl.alt} />
+                      </a>
+                    </SponsorCard>
```

Comment:
```suggestion
                    <div key={spl.link} className="m-4 text-secondary bg-white">
                      <a href={spl.link} target={spl.target} rel={spl.rel}>
                      <img src={spl.src} style={{ height: spl.imageHeight }} className={spl.className} alt={spl.alt} />
                      </a>
                    </div>
```

Reply:
SponsorCard doesn't really do anything, unless you have plans for it.

Reply:
check my comment above for the explanation

Reply:
what do you mean? I changed the `SponsorCard` to`div` and nothing changed. It seems to be working fine